server/apiCalls/transformers/memos.test.js

```diff
@@ -188,6 +200,9 @@ describe('Memos functions to count memos for one school', () => {
           "connectedPrograms": "TTGTM-2",
           "courseCode": "SF1625",
           "courseMemoInfo": {
+            "applicationCodes": [
+              "1",
+            ],
             "courseCode": "SF1625",
             "isPdf": false,
             "ladokRoundIds": [
```

Comment:
`ladokRoundIds`  has to be removed to be sure tests working properly

Reply:
Removed